packages/__docs__/build-docs.mjs

```diff
@@ -0,0 +1,4 @@
+// scripts must be built from .ts files before this is run
+import {buildDocs} from './lib/build-docs.mjs'
+
+buildDocs()
```

Comment:
was this file created in windows? I think this file ending is not compatible with unix EOF. I think editing this file in macos should solve this

Reply:
also the same file name with `lib/build-docs.mjs` is a bit confusion. I'm wondering if there's that file already, can't that one be run instead?

Reply:
There is a way not to create a new file and not use fancy syntax in `package.json` but its also ugly: https://stackoverflow.com/a/75760840 What do you think?

Reply:
fixed it, now it doesn't need an extra file